tools/VerifyCopyrightHeaders.ps1

```diff
@@ -3,12 +3,12 @@ Param(
 )
 
 $copyrightHeaderText = (
-    "Copyright (c) Microsoft Corporation.",
+    "Copyright (c) Microsoft Corporation and Contributors.",
     "Licensed under the MIT License."
 )
 
 $include = ('*.cs', '*.cpp', '*.h', '*.idl', '*.xaml')
-$exclude = [RegEx]'\\dev\\Detours\\'
+$exclude = [RegEx]'\\dev\\Detours\\|\\BuildOutput\\|\\obj\\|\\localpackages\\|\\packages\\|\\specs\\|\\temp\\|\\Debug\\|\\Release\\'
```

Comment:
Missing exclusions:

* docs
* x86, x64, arm64 (e.g. `C:\source\repos\windowsappsdk\dev\PushNotifications\PushNotificationsLongRunningTask.ProxyStub\x64\Debug\NotificationsLongRunningProcess_h.h`)
* temp -- probably should be excluded

Are .dirs traversed? If so thyen .config, .github and .user probably should be excluded

Reply:
- docs doesn't have any of the included file types.
- x86/x64/arm64 are taken care of with the debug/release exclusions
- temp is already excluded